arch/rootfs/arm64/.config

```diff
@@ -437,7 +437,7 @@ BR2_TARGET_TZ_INFO=y
 BR2_TARGET_TZ_ZONELIST="default"
 BR2_TARGET_LOCALTIME="PRC"
 BR2_ROOTFS_USERS_TABLES=""
-BR2_ROOTFS_OVERLAY=""
+BR2_ROOTFS_OVERLAY="$(BR2_EXTERNAL_OVERLAY)"
```

Comment:
What is `BR2_EXTERNAL_OVERLAY`?

Reply:
```shell
make BR2_EXTERNAL_OVERLAY=$(pwd)/board/overlay rootfs-arm64-build
```

to see https://buildroot.org/downloads/manual/manual.html#outside-br-custom.